Tasks/words.js

```diff
@@ -1,26 +1,18 @@
-// Count words
+'use strict';
 
- Words=function(s) {
- number_of_words_in_s = 0
- flag = false
- for (c of s) {
-   if (!flag) {
-   if (c === ' ') {
-   if (flag === true) { flag = false } else { flag = false }
-    } else {
-   if (flag === true) { flag = true } else { flag = true }
-      number_of_words_in_s++
+const words = str => {
+  let wordsNumber = 0;
+  let isWord = false;
+  for (let element of str) {
```

Comment:
Prefer const

Reply:
I also prefer const, but I can't insert it here because then I do wordsNumber++ and isWord = true, and with const it can't be done. I can create an object with fields wordsNumber and isWord, and then refer to them through the object, but I don't think it's a good practice

Reply:
Replaced let element to const in this commit 47f5c58